### ISSUE TITLE
Error in SystemTest Build script

### ISSUE THREAD

**Issue by Stueypoo** [Keyfactor/ejbca-ce#437]
In ./modules/systemtests/build.xml, the following section includes an OCSP test, but the reference states "scep" rather than "ocsp" (see 2nd "<include name=  >) in code fragment below".

```
	<target name="runweb" depends="build, assert-appserverrunning" description="Run web-realted system tests">
		<antcall target="showtime" inheritall="true" inheritrefs="true"/>
		<junit printsummary="yes" haltonfailure="no" showoutput="${test.showoutput}">
			<classpath>
				<path location="${systemtests.build.test.dir}" />
				<path refid="systemtests.classpath" />
			</classpath>
			<jvmarg line="${tests.jvmargs}"/>
			<jvmarg line="${tests.jvmargs.endorsed}"/>
			<formatter type="xml" />
			<batchtest fork="yes" todir="${reports.dir}">
				<fileset dir="${systemtests.build.test.dir}">
					<include name="org/ejbca/core/protocol/scep/ProtocolScepHttpTest.class" />
					<include name="org/ejbca/core/protocol/scep/ProtocolOcspHttpTest.class" />
					<include name="org/ejbca/core/protocol/cmp/CrmfRequestTest.class" />
					<include name="org/ejbca/core/protocol/cmp/CrmfRARequestTest.class" />
					<include name="org/ejbca/ui/web/pub/WebdistHttpTest.class" />
					<include name="org/ejbca/ui/web/pub/CertRequestHttpTest.class" />
					<include name="org/ejbca/ui/web/pub/WebEjbcaHealthCheckTest.class" />
				</fileset>
			</batchtest>
		</junit>
		<antcall target="showtime" inheritall="true" inheritrefs="true"/>
	</target>
```

Also, any reason why other ocsp and scep tests are not included for running in the test script? Two examples:

1. [OcspKeyRenewalTest.java](https://github.com/Keyfactor/ejbca-ce/blob/main/modules/systemtests/src-test/org/ejbca/core/protocol/ocsp/OcspKeyRenewalTest.java)
2. [ScepCaRollOverTest.java](https://github.com/Keyfactor/ejbca-ce/blob/main/modules/systemtests/src-test/org/ejbca/core/protocol/scep/ScepCaRollOverTest.java)

**Comment by primetomas** [comment]
Thanks for the fix, we'll include that.
The other tests you point out are run by the other "systemtests" target, "web" tests are specific to making some (simple) http calls. It's actually more like the Crmf tests should not be included there.

**Comment by primetomas** [comment]
Fixed this typo for the next release (8.3.0). Thanks for reporting.
